cmd/update-notifier.go

```diff
@@ -60,7 +74,7 @@ func colorizeUpdateMessage(updateString string, newerThan time.Duration) string
 
 	// Box cannot be printed if terminal width is small than maxContentWidth
 	if maxContentWidth > termWidth {
-		return "\n" + line1InColor + "\n" + line2InColor + "\n" + "\n"
+		return fmt.Sprint("\n", line1InColor, "\n", line2InColor, "\n\n")
```

Comment:
I thought doing so many concatenations looking inefficient, so I used `fmt.Sprint` - but some simple benchmarking later, it appears the inefficiency is only apparent for a larger number of strings. I switched back to the `+` operation in this code. 

Also here, `strings.Join` is not making it more readable as there are `"\n"` near the beginning and end and extra and there are just two lines with content. So `+` seems fine here.